src/lib/Microsoft.Health.Fhir.Ingest/Config/ResourceIdentityServiceType.cs

```diff
@@ -0,0 +1,28 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Ingest.Config
+{
+    /// <summary>
+    /// Resource Identity Service Type.
+    /// </summary>
+    public enum ResourceIdentityServiceType
```

Comment:
Please make sure this resolves properly when changed to an enumeration.  Specifically you want to make sure builder.Services.Configure<ResourceIdentityOptions>(config.GetSection("ResourceIdentity")); in MeasurementFhirImportExtensions is properly resolving the type.

Reply:
Yes. I've confirmed the string value can be resolved to this enum correctly with all our templates.